documentation/src/main/asciidoc/topics/proc_displaying-metrics-st-operation.adoc

```diff
@@ -0,0 +1,41 @@
+[id='displaying-metrics-st-operationadoc_{context}']
+= Displaying metrics during a state transfer operation
+
+You can export time metrics for clustered caches that {brandname} redistributes across nodes. 
+
+A state transfer operation occurs when a clustered cache topology changes, such as a node joining a cluster, so the cache's data must be redistributed to all nodes in a cluster. 
+
+[NOTE]
+====
+You cannot perform a state transfer operation in invalidation mode, because this mode restricts database write operations after state changes occur. 
+====
+
+{brandname} generates time metrics that are compatible with the REST API and the JMX API. 
+
+Consider accessing metrics through the REST endpoint. 
+This endpoint uses a `StateTransferManager` attribute to query the following methods:
+
+* `getInflightSegmentTransferCount()`, which returns the number of in-flight requested segments.
+* `getInflightTransactionalSegmentCount()`, which returns the number of in-flight transactional requested segments.
+
+[NOTE]
+====
+In scattered cache mode, you cannot view time metrics because this mode is not exposed in the server configuration file. 
```

Comment:
I'll check this.

Reply:
Just checked this. If the cache is created with the statistics enabled it will expose these metrics. I think we can remove this note.

Reply:
Removed.